server/strong-oracle.js

```diff
@@ -0,0 +1,49 @@
+var settings = {};
```

Comment:
that was just an example, not the main point.

the oracle tests are 3 copies of the same code, and the memcache(d) is 2 copies. You can refactor by making a base file that exports a function that just takes the module as an argument. Makes things much shorter. Basically, avoid cut-n-paste, it feels fast at the moment, but always comes back to haunt you or some future maintainer.